modules/richards/tests/gravity_head_2/tests

```diff
@@ -171,4 +171,43 @@
     rel_err = 1e-5
     abs_zero = 1e-4
   [../]
+
+  [./except01]
+    type = 'RunException'
+    input = 'gh01.i'
+    cli_args = 'Materials/rock/viscosity="1"'
+    expect_err = 'There are 2 Richards fluid variables, so you need to specify this number of viscosities, relperm_UO, seff_UO, sat_UO, density_UO, SUPG_UO'
+  [../]
+  [./except02]
+    type = 'RunException'
+    input = 'gh01.i'
```

Comment:
If these tests have an output block they will clobber each other. You have a few choices of making this work in parallel. 1. You can add prereqs to each of these blocks to force serialization
2. You can remove the `[Outputs]` block from files that will be used for `expect_err` tests
3. You can provide a `cli_args` to change the `file_base` in each block so they don't clobber each other.

This is a good idea, even if you aren't planning on inspecting the output. It's just good practice in case you decide to later.